src/XFExpandableListView/Controls/ExpandableListView.cs

```diff
@@ -206,16 +209,19 @@ public virtual async Task Expand(int position)
             /* Hard operations must not affect the UI Thread */
             await Task.Run(() =>
             {
-                foreach (var item in expandableGroup)
+                Device.BeginInvokeOnMainThread(() =>
```

Comment:
No need to create a new thread that will be executed on the main thread. Just get it out from the Task.Run

Reply:
Removed